dms/models/dms_file.py

```diff
@@ -179,6 +173,7 @@ def check_access_token(self, access_token=False):
         invisible=True,
         ondelete="cascade",
     )
+    storage_id_save_type = fields.Selection(related="storage_id.save_type", store=False)
```

Comment:
This change does not exist in #102. Is it really needed?

Reply:
Sorry, it's not necessary because exists in `abstract.dms.mixin`. Thanks!